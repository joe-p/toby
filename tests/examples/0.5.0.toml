date = 1979-05-27
time = 07:32:00

plus-infinity = +inf
# infinity
sf1 = inf  # positive infinity
sf2 = +inf # positive infinity
sf3 = -inf # negative infinity

# not a number
sf4 = nan  # actual sNaN/qNaN encoding is implementation specific
sf5 = +nan # same as `nan`
sf6 = -nan # valid, actual encoding is implementation specific

# hexadecimal with prefix `0x`
hex1 = 0xDEADBEEF
hex2 = 0xdeadbeef
hex3 = 0xdead_beef

# octal with prefix `0o`
oct1 = 0o01234567
oct2 = 0o755 # useful for Unix file permissions

# binary with prefix `0b`
bin1 = 0b11010110

# Dotted keys
physical.color = "orange"
physical.shape = "round"
site."google.com" = true

a.b.c = 1
a.b.d = 2


[table]
a.b.c = 1
a.b.d = 2
a.c = 3
