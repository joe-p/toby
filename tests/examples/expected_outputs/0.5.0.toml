date = 1979-05-27
time = 07:32:00.999
plus-infinity = Infinity

#infinity
sf1 = Infinity #positive infinity
sf2 = Infinity #positive infinity
sf3 = -Infinity #negative infinity

#not a number
sf4 = NaN #actual sNaN/qNaN encoding is implementation specific
sf5 = NaN #same as `nan`
sf6 = NaN #valid, actual encoding is implementation specific

#hexadecimal with prefix `0x`
hex1 = 0xdeadbeef
hex2 = 0xdeadbeef
hex3 = 0xdeadbeef

#octal with prefix `0o`
oct1 = 0b1234567
oct2 = 0b755 #useful for Unix file permissions

#binary with prefix `0b`
bin1 = 0b11010110

#Dotted keys
physical.color = "orange"
physical.shape = "round"
site."google.com" = true
a.b.c = 1
a.b.d = 2
[table]
a.b.c = 1
a.b.d = 2
a.c = 3
